### DISCUSSION TITLE
Bootstrap github workflows usage

### DISCUSSION THREAD

**Discussion by marioja** [twbs/bootstrap#37520] [General] [answered]
I looked at how you use Hugo to generate your documentation and website and I would love to duplicate it.  However for the love of me, I am unable to figure out how this is all glued together.   Specifically, I do not understand how your changes in branch v5.2.2 to say  site/layouts/partials/home/masthead.html becomes live at http://getbootstrap.com/ or how a change to site/content/docs/5.2/components/navbar.md becomes live at https://getbootstrap.com/docs/5.2/components/navbar/.

I understand that site/layouts/_default/home.html use hugo partials on masthead.html but I do not undertand the glue or how this is all linked.  Now, please forgive me if I missed some hugo documentation or something about github actions as I am no guru in neither of those.  I see that the pages-build-deployment github builtin actions last ran a couple months ago on your repo and I saw that you have a docs.yml workflow which builds the docs but I do not understand:

1. how do you use pages-build-deployment to deploy your site when this action cannot be modified and out of the box does not use hugo?
2. why is your repo not in an organization bootstrap and the URL is not bootstrap.github.io?
3. What happens to the output from the docs workflow, I do not see it committed to git, how does that make it to your live website.
Any tips or light you can shed on how you do it will be greatly appreciated.  Thanks and I love and truly appreciate your great work.

**Comment by mdo** [comment, marked as answer]
We build the docs and push them manually to the `gh-pages` branch. We host our docs on GitHub Pages. Check out our Icons project, that one automatically builds and pushes via Actions.
